python/ray/serve/_private/common.py

```diff
@@ -755,6 +759,77 @@ class CreatePlacementGroupRequest:
     "Use handle.options(_by_reference=True) to enable it."
 )
 
+
+class AutoscalingStatus(str, Enum):
+    UPSCALE = "AUTOSCALING_UPSCALE"
+    DOWNSCALE = "AUTOSCALING_DOWNSCALE"
+    STABLE = "AUTOSCALING_STABLE"
+
+
+class DecisionRecord(BaseModel):
+    timestamp_str: str
+    current_num_replicas: int
+    target_num_replicas: int
+    reason: str
+
+
+class DeploymentSnapshot(BaseModel):
+    timestamp_str: str
+    app: str
+    deployment: str
+    current_replicas: int
+    target_replicas: int
+    min_replicas: Optional[int]
+    max_replicas: Optional[int]
+    scaling_status: str
+    policy_name: str
+    look_back_period_s: Optional[float]
+    queued_requests: Optional[float]
+    total_requests: float
```

Comment:
this is to standardize the naming across the repo: queued_requests, running_requests and ongoing_requests. ongoing_requests = queued_requests + running_requests

```suggestion
    ongoing_requests: float
```